03_Modules/Monitoring/test/module/MonitoringService.test.ts

```diff
@@ -0,0 +1,63 @@
+import { IVariableMonitoringRepository } from '@citrineos/data';
+import { MonitoringService } from '../../src/module/MonitoringService';
+import {
+  ClearMonitoringResultType,
+  ClearMonitoringStatusEnumType,
+} from '@citrineos/base';
+import { aClearMonitoringResult } from '../providers/Monitoring';
+
+describe('MonitoringService', () => {
+  let mockVariableMonitoringRepository: jest.Mocked<IVariableMonitoringRepository>;
+  let monitoringService: MonitoringService;
+
+  beforeEach(() => {
+    mockVariableMonitoringRepository = {
+      rejectVariableMonitoringByIdAndStationId: jest.fn(),
+    } as unknown as jest.Mocked<IVariableMonitoringRepository>;
+
+    monitoringService = new MonitoringService(mockVariableMonitoringRepository);
```

Comment:
since we are just initializing the class here, I think you can just define these outside of `beforeEach` right?

Reply:
indeed i think it is like this right now only because the dependencies themselves are also declared in the beforeEach 🤔 perhaps something we can look into simplifying down the line so we don't have to remake the service every time... unless jest doesn't let us do that easily 😅